funnel/views/event.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+from flask import redirect, render_template, url_for, flash
+from coaster.views import load_models
+from sqlalchemy.exc import IntegrityError
+from .. import app, lastuser
+from ..models import (db, Profile, ProposalSpace, ProposalSpaceRedirect, Participant, Event, TicketType, TicketClient)
+from baseframe import forms
+from baseframe import _
+from baseframe.forms import render_form
+from ..forms import EventForm, TicketTypeForm, ParticipantBadgeForm, TicketClientForm
+from ..jobs import import_tickets
+
+
+@app.route('/<space>/admin', methods=['GET', 'POST'], subdomain='<profile>')
+@lastuser.requires_login
+@load_models(
+    (Profile, {'name': 'profile'}, 'g.profile'),
+    ((ProposalSpace, ProposalSpaceRedirect), {'name': 'space', 'profile': 'profile'}, 'space'),
+    permission='admin')
+def admin(profile, space):
+    attendee_sync_form = forms.Form()
+    if attendee_sync_form.validate_on_submit():
+        for ticket_client in space.ticket_clients:
+            if ticket_client and ticket_client.name == u'explara':
+                import_tickets.delay(app.config['ENV'], ticket_client.id)
```

Comment:
Don't provide `init_for` env to the background job. We use separate mechanisms in RQ to do that when the worker starts, not when the job starts.